src/test/incremental/hashes/call_expressions.rs

```diff
@@ -183,11 +171,11 @@ pub fn change_to_ufcs() {
     s.method1('x', true);
 }
 
+// FIXME(vitiral): why would this change anything, doesn't the Mir/Hir expand this
+// sort of stuff?
```

Comment:
I left a comment on it (not a FIXME). It seems wrong to me that this would happen from what I have read (which is all beginner material/new-rustacean).